types/osf-api.d.ts

```diff
@@ -1,4 +1,4 @@
-/* eslint-disable no-use-before-define,camelcase */
+/* eslint-disable camelcase */
 
 import * as JSONAPI from 'jsonapi-typescript';
```

Comment:
why not use camelcase here?

Reply:
Wa just following [the instructions](https://github.com/mike-north/jsonapi-typescript#how-to-use-this). :wink:

Reply:
The reason for `eslint-disable camelcase` is because this definition file types the API response document, which uses snake_case, exclusively.

Reply:
But since this is living in the Javascript-side of things we can define whatever we want, no?

Reply:
Sorry, when you said "here" were you referring to the `JSONAPI` namespace (which doesn't violate the `camelcase` rule) or to this whole file (which has keys like `per_page` because it's typing the JSON document as returned by the API)?

Reply:
nah I was talking about the json-api underscores n stuff. Not really a huge deal.